guides/common/modules/con_template-writing-reference.adoc

```diff
@@ -18,7 +18,7 @@ For more information, see {ProvisioningDocURL}creating-partition-tables_provisio
 
 ifndef::orcharhino[]
```

Comment:
@maximiliankolb I wonder why this is still here. Given `ManagingConfigurationsPuppetDocURL` is upstream, doesn't Orcharhino have this?

Reply:
Thanks for your hint! See PR https://github.com/theforeman/foreman-documentation/pull/1814